### PR TITLE
reformat bitwise_and

### LINKED ISSUE #1374: bitwise_and

### PR DESCRIPTION
**Pull request by jackhong12**
resolve #1374

### CODE REVIEW

**Comment by jackhong12** [comment]
Hi @chie2727. Could you review my PR and give me some feedback? Thanks a lot!

**Review by chie2727 — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/elementwise.py [hunk 1]
```diff
@@ -454,6 +454,113 @@ def bitwise_and(
...
+
+    >>> x = ivy.Container(a=ivy.array([1, 2, 3]), b=ivy.array([4, 5, 6]))
+    >>> y = ivy.Container(a=ivy.array([7, 8, 9]), b=ivy.array([10, 11, 11]))
+    >>> z = ivy.bitwise_and(x, y)
+    >>> print(z)
+    {
+        a: ivy.array([1, 0, 1]),
+        b: ivy.array([0, 1, 2])
+    }
+
```
[the remark this answers is not in the document]
Comment (jackhong12) [reply ivy/functional/ivy/elementwise.py:529]: done!

**Comment by jackhong12** [comment]
I have added the example that uses a mix of `ivy.Array` and `ivy.Container` inputs.

**Review by chie2727 — APPROVED** [review]
Thanks for making the changes! Everything is looking good so I'll merge your edits in

**Comment by jackhong12** [comment]
Thanks!

**Pull request merged**